pkg/errors/errors.go

```diff
@@ -105,7 +105,7 @@ var (
 	ErrDivertNotSupported = fmt.Errorf("the 'divert' field is only supported in clusters that have Okteto installed")
 
 	// ContextIsNotOktetoCluster raised if the cluster connected is not managed by okteto
-	ErrContextIsNotOktetoCluster = fmt.Errorf("this command is only available on Okteto Cloud or Okteto Enterprise")
+	ErrContextIsNotOktetoCluster = fmt.Errorf("this command is only available in clusters where Okteto is installed.\n Follow this link to know more about configuring Okteto at your cluster: https://www.okteto.com/docs/self-hosted/")
```

Comment:
It think it's ok, it should be updated for all. A less cloud-based message makes sense with our current approach. @pchico83 can you also confirm?

Reply:
I saw the error was "already there", so instead of creating an error with the same "message" i completed with the URL and rephrase it. for `okteto destroy --all` i created a fmt.Errorf, but i think this should be applied also.

Reply:
yeah, just wanted to make sure we are ok on changing the message in any place where it is being used. Makes sense for me